modules/core/csv/src/Normalizer/EntityReferenceFieldItemNormalizer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\farm_csv\Normalizer;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItemInterface;
+use Drupal\serialization\Normalizer\EntityReferenceFieldItemNormalizer as CoreEntityReferenceFieldItemNormalizer;
+
+/**
+ * Normalizes entity reference fields for farmOS CSV exports.
+ */
+class EntityReferenceFieldItemNormalizer extends CoreEntityReferenceFieldItemNormalizer {
+
+  /**
+   * The supported format.
+   */
+  const FORMAT = 'csv';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize($field_item, $format = NULL, array $context = []): array|string|int|float|bool|\ArrayObject|NULL {
+
+    // Attempt to load the referenced entity.
+    /** @var \Drupal\Core\Entity\EntityInterface $entity */
+    if ($entity = $field_item->get('entity')->getValue()) {
+
+      // Return content entity labels, if desired.
+      if ($entity instanceof ContentEntityInterface && !empty($context['content_entity_labels'])) {
```

Comment:
Could we do strict comparisons for these context variables we're checking? I think this makes it easier to read and is more explicit what the expected value should be. `!empty()` is more compact but also less strict and prone to causing errors more general situations (when a `0` is passed) so it shouldn't be relied upon as a default convention IMO.

And as my AI friend said when asked about using `empty()` vs `isset()` + strict comparison check:

> In any case, always choose clarity and avoid potential errors even if the code is a bit longer. Readability and avoidance of errors are often more critical than writing the most compact code.

```suggestion
      if ($entity instanceof ContentEntityInterface && isset($context['content_entity_labels']) && $context['content_entity_labels'] === TRUE) {
```

Reply:
Included in a commit